compiler/src/iree/compiler/Codegen/LLVMCPU/test/pipeline_tests.mlir

```diff
@@ -562,6 +562,7 @@ hal.executable private @unsupported_ukernel_fallback_to_vectorization {
 //
 //   * CPUBufferOpsTileAndVectorize
 //   * CPUDoubleTilingPeelingExpert
+//   * CPUDoubleTilingExpert
```

Comment:
Enabling SSVE for `CPUDoubleTilingExpert` seems like a separate change to adding the flag to disable tiling.

Reply:
https://github.com/openxla/iree/pull/16218